app/models/image.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class Image < ApplicationRecord
+  belongs_to :document
+  belongs_to :blob, class_name: "ActiveStorage::Blob" # rubocop:disable Rails/InverseOf
+
+  WIDTH = 960
+  HEIGHT = 640
+
+  def crop_variant
+    crop = "#{crop_width}x#{crop_height}+#{crop_x}+#{crop_y}"
+    blob.variant(
```

Comment:
I think we should add `colorspace: "sRGB"` here to normalise any colour profiles.

Reply:
Thanks - that feels like something that'd be better served [pre-cropping](https://github.com/alphagov/content-publisher/blob/e6cd962cae1cf79b5c764933da20b52754bf6293/app/services/uploaded_image_service/image_normaliser.rb#L20-L27) 

I don't know much/anything about colour profiles - is that stored in exif data or some different way? and I'm not too sure how to test whether adding something will have an effect.

Reply:
That does look like a better place, but I wasn't sure that the it could be added there.